pom.xml

```diff
@@ -124,6 +125,7 @@
             </plugin>
             <plugin>
                 <artifactId>maven-source-plugin</artifactId>
+                <version>2.4</version>

```

Comment:
Plugin versions were removed to prevent security issues when specifying older versions. The "missing" version message only appears in older versions of Maven. Not specifying the version, ensures you get the latest available.